policyengine_us/parameters/gov/states/ar/tax/income/exemptions/exemptions.yaml

```
@@ -1,21 +1,12 @@
-description: Arkansas accounts for these exemptions when calculating the state adjusted gross income.
+description: Arkansas exempts these sources from state adjusted gross income.
```

Please also update the issue and PR title
```suggestion
description: Arkansas exempts these sources from state gross income.
```